dealership/vehicles.py

```diff
@@ -1,15 +1,38 @@
 class Vehicle(object):
+
+    sale_multiplier = 1.0
+    purchase_multiplier = 1.0
+
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
+
+    def sale_price(self):
+        return self.base_price * self.sale_multiplier
+
+    def purchase_price(self):
+        return self.sale_price() - self.purchase_multiplier * self.miles
 
 
 class Car(Vehicle):
-    pass
+    sale_multiplier = 1.2
+    purchase_multiplier = 0.004
+    interest_multiplier = 1.07
+    lease_multiplier = 1.2
```

Comment:
I like the way you have made these class attributes - this seems cleaner than the way I did it.